src/components/app.scss

```diff
@@ -23,69 +25,31 @@ BODY {
     flex-basis: auto;
   }
 
-  .main-controls-and-waves-container {
-    // FUTURE: consider making this be of variable height, based on the viewport
-    margin: 0 $margin;
+  .non-header-container {
+    height: 100%;
+    margin: 6px;
+    padding: 3px 3px 6px 3px;
+    border: 2px solid $darkBorderColor;
+    border-radius: $borderRadius;
   }
 
-  .speed-controls {
-    margin-bottom: -2px;
-    height: 50px;
-    padding: 4px 15px 2px 15px;
+  .main-controls-and-waves-container {
+    height: calc(50% - 30px - 15px);
```

Comment:
Minor thing, but if 30px and 15px are important for the layout to look correct and related to some other styles, I guess we could extract these values as variables and use them in both places.